src/port1.0/portutil.tcl

```diff
@@ -3309,7 +3309,8 @@ proc _check_xcode_version {} {
                 ui_warn "You downloaded Xcode from the Mac App Store but didn't install it. Run \"Install Xcode\" in the /Applications folder."
             }
             if {[tbool use_xcode]} {
-                return -code error "This port requires the full Xcode installation, which was not found on your system. You can install Xcode from the Mac App Store or https://developer.apple.com/xcode/"
+                ui_error "Port ${subport} requires the full Xcode installation, which was not found on your system. You can install Xcode from the Mac App Store or https://developer.apple.com/xcode/"
```

Comment:
I think "a full Xcode installation" would sound more natural than "the full Xcode installation".

Reply:
Changed